prime-router/settings/organizations-local.yml

```diff
@@ -308,6 +308,8 @@
           reportingFacilityId: 36DSMP9999
           nameFormat: OHIO
           suppressHl7Fields: OBX-23-11
+          # turn off UNK and ASKU for this field
+          useBlankInsteadOfUnknown: patient_race
         timing:
```

Comment:
Some of the receivers want to not get `ASKU` or `UNK` for certain LOINC fields. Rather than create a property for each of those coded fields, `useBlankInsteadOfUnknown` takes a list of fields and then blanks out `ASKU` and `UNK`.

Reply:
I'm very mildly concerned that the rules for what happens to a particular element of data are getting scattered around in various places.   Something to think about consolidating somehows, someday.

Reply:
I am concerned about it as well. I'm not super happy with injecting all of these changes this way, but I am not sure if we have another way to do manage it, unless we return back to using a schema and setting them all the Custom translations.